package.json

```diff
@@ -27,7 +29,14 @@
     "autoprefixer": "^10.4.7",
     "eslint": "8.18.0",
     "eslint-config-next": "12.1.6",
+    "husky": ">=6",
+    "lint-staged": ">=10",
     "postcss": "^8.4.14",
+    "prettier": "^2.7.1",
     "tailwindcss": "^3.1.4"
+  },
+  "lint-staged": {
```

Comment:
This defines the configuration for `lint-staged`. 

Anytime a contributor runs `git commit`, `lint-staged` will check this configuration to determine which commands to run (if any) for each staged file that is part of the commit.